Buttplug.Apps.GameVibrationRouter.GUI/MainWindow.xaml.cs

```diff
@@ -70,6 +74,10 @@ public MainWindow()
             ButtplugTab.AddDevicePanel(_bpServer);
             ButtplugTab.SelectedDevicesChanged += OnSelectedDevicesChanged;
 
+            _vibeTab = new VibeConfigTab();
+            _vibeTab.MultiplierChanged += MultiplierChanged;
+            ButtplugTab.SetOtherTab2("Vibe Config", _vibeTab);

```

Comment:
Hmm. It kinda feels like we should maybe create a system to push new tabs versus just adding tabs manually like this, but I guess it works for now (and I vaguely remember having some issue trying to implement that when I made the vibration graph tab). 

If I ever get huffy about this in the future, remind me I made this comment.

Reply:
Yeah I was kinda thinking the same but I'm still not that great at WPF and didn't really feel too comfortable trying a major(ish) rewrite when I noticed how the other one was added... :/